src/schema/v2/quizArtworkConnection.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { ArtworkType } from "schema/v2/artwork"
+import {
+  connectionWithCursorInfo,
+  paginationResolver,
+} from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+
+export const QuizArtworkType = new GraphQLObjectType<any, ResolverContext>({
+  name: "QuizArtwork",
+  fields: {
+    artworkId: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ artwork_id }) => artwork_id,
+    },
+    interactedAt: {
+      type: GraphQLString,
+      resolve: ({ interacted_at }) => interacted_at,
+    },
+    position: {
+      type: new GraphQLNonNull(GraphQLInt),
+      resolve: ({ position }) => position,
+    },
+    artwork: {
+      type: new GraphQLNonNull(ArtworkType),
+      resolve: ({ artwork }) => artwork,
+    },
+  },
+})
+
+export const QuizArtworkConnectionType = connectionWithCursorInfo({
+  nodeType: QuizArtworkType,
+}).connectionType
+
+export const quizArtworkConnection: GraphQLFieldConfig<any, ResolverContext> = {
+  type: QuizArtworkConnectionType,
+  args: pageable({
+    page: { type: GraphQLInt },
+    size: { type: GraphQLInt },
+  }),
+  resolve: ({ quiz_artworks }, args) => {
+    const { page, size, offset } = convertConnectionArgsToGravityArgs({
+      first: args?.first | 16,
```

Comment:
This isn't a logical `or` but is a bitwise operation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR

Looking for `||`

Reply:
Thank you.